robosuite/utils/control_utils.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import robosuite.utils.transform_utils as trans
+import scipy.linalg
+
+
+def nullspace_torques(mass_matrix, nullspace_matrix, initial_joint, joint_pos, joint_vel):
+    joint_kp = 10
```

Comment:
should this be an argument with a default value?

Reply:
@cremebrule following up on this

Reply:
yes! good call